main/py-pillow/APKBUILD

```diff
@@ -44,19 +44,7 @@ _py() {
 	install_if="$pkgname=$pkgver-r$pkgrel $python"
 
 	cd "$builddir"
-	$python setup.py install --prefix=/usr --root="$subpkgdir" || return 1
-
-	# The scripts are packages in -doc, they should not be here.
-	rm -r "$subpkgdir"/usr/bin
-}
-
-doc() {
```

Comment:
I think that we still should ship the docs. In #3878 I started with this:

```bash
doc() {
	pkgdesc="$pkgdesc (documentation)"
	local docdir="$subpkgdir/usr/share/doc/$pkgname"

	cd "$builddir"/docs
	mkdir -p "$docdir"
	make html
	cp -R _build/html "$docdir"
}
```

Will require Sphinx.

Reply:
@fabaff thank you very much for the feedback. Building the html docs for `py-pillow` requires `py-sphinx` and `py-packaging`. The later is in the community repository and travis cannot pull it in as a dependency.